layouts/shortcodes/kat-button

```diff
@@ -1,3 +1,2 @@
-<div id="my-panel" data-katacoda-ondemand="true" data-katacoda-port="30000" data-katacoda-env="minikube" data-katacoda-command="start.sh" data-katacoda-ui="panel"></div>
-<script src="https://katacoda.com/embed.js"></script>
+<script defer src="https://katacoda.com/embed.js"></script>
```

Comment:
If we wanted to, we could move this into `layouts/partials/hooks/head-end.html`

I didn't test that.